docs/source/getting_started/quick_start.rst

```diff
@@ -223,20 +221,20 @@ f. Set Up Resources for Every Module
 
    .. note :: * You can run the workflow with fewer cores, if the above resources are not available. This increases runtime respectively.
 
-        * Memory is provided in MB in the resources tab. Running Nanoscope with less memory than indicated in the table above is possible, but you may run into out-of-memory issues especially for larger molecules.
+        * Memory is provided in MB in the ``Resources`` tab. Running Nanoscope with less memory than indicated in the table above is possible, but you may run into out-of-memory issues especially for larger molecules.
 
-        * Further information on resources is provided in the :ref:`user_guide_settings` section.
```

Comment:
I'm fine with the new note below, but why kill the link to the settings description?

Reply:
because it is not provided there

Reply:
True